governance/overview.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Overview
+description: As a Polkadot parachain, Moonbeam will use an on-chain governance system, allowing for a stake-weighted vote on public referenda.
+---
+
+# Governance in Moonbeam
+
+![Governance Moonbeam Banner](/images/governance/governance-overview-banner.png)
+
+## Introduction
+
+Moonbeam is a decentralized network that will be governed by a community of stakeholders, including core developers, application developers, collators, token holders, users, partners, and other contributors. 
```

Comment:
"partners" feels funny here. It makes me think of companies partnering with other companies. Of course the company PureStake has partners, but does the Moonbeam network have partners? Are there any partners that don't fit into the other categories you've mentioned?